scalyr_agent/scalyr_client.py

```diff
@@ -986,6 +988,45 @@ def _calculate_per_thread_extra_bytes():
     return result
 
 
+def _calculate_per_log_extra_bytes():
+    """Calculates how many extra bytes are added to the serialized form of the threads JSON array
+    when adding a new thread, excluding the bytes for serializing the thread id and name themselves.
+
+    This is used below by the PostFixBuffer abstraction to help calculate the number of bytes the serialized form
+    of the PostFixBuffer will take, without having to actually serialize it.  It was found that doing the heavy
+    weight process of serializing it over and over again to just get the size was eating too much CPU.
+
+    @return: An array of two int entries.  The first entry is how many extra bytes are added when adding the
+        first thread to the threads JSON array and the second is how many extra bytes are added for all subsequent
+        threads.  (The number differences by at least one due to the need for a comma to be inserted).
```

Comment:
s/The number differences/The number differs/

Reply:
Fixed where I copied it from as well.